lib/cryptoexchange/exchanges/bittrex/services/market.rb

```diff
@@ -10,21 +10,22 @@ def supports_individual_ticker_query?
 
         def fetch(market_pair)
           output = super(ticker_url(market_pair))
-          adapt(output)
+          adapt(output, market_pair)
         end
 
         def ticker_url(market_pair)
           base = market_pair.base
           target = market_pair.target
-          "#{Cryptoexchange::Exchanges::Bittrex::Market::API_URL}/public/getmarketsummary?market=#{base}-#{target}"
+          # Bittrex pair has BTC comes first, when BTC is typically a Target not a Base
+          "#{Cryptoexchange::Exchanges::Bittrex::Market::API_URL}/public/getmarketsummary?market=#{target}-#{base}"
```

Comment:
@tmlee Looking at the response of `https://bittrex.com/api/v1.1/public/getmarketsummary?market=btc-ltc`

```
{
    "success": true,
    "message": "",
    "result": [
        {
            "MarketName": "BTC-LTC",
            "High": 0.011225,
            "Low": 0.01054107,
            "Volume": 108671.50400962,
            "Last": 0.01082199,
            "BaseVolume": 1186.30109815,
            "TimeStamp": "2017-08-20T05:07:00.463",
            "Bid": 0.01082199,
            "Ask": 0.010822,
            "OpenBuyOrders": 1168,
            "OpenSellOrders": 6783,
            "PrevDay": 0.01100987,
            "Created": "2014-02-13T00:00:00"
        }
    ]
}
```

I think we should use `Volume` instead of `BaseVolume`?